### PR TITLE
OEA-419, fix the problem when operation reschedule between anaesthetist required or not

### PR DESCRIPTION
**Pull request by Zhaolian-Zhou**

### CODE REVIEW

File: protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php [hunk 1]
```diff
@@ -1199,15 +1197,15 @@ public function schedule($booking, $operation_comments, $session_comments, $oper
                 }
             }
         }
-
+        $this->booking->session = $session;
```
Comment (apigram) [inline protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php]: Does this actually work? I can't imagine that would update the IDs and would therefore not be linked correctly.

Comment (Zhaolian-Zhou) [reply protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php]: Currently it seems work, but I suspect as well. Maybe keep observing this part.

Comment (apigram) [reply protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php]: @marshl What are your thoughts on this?

Comment (marshl) [reply protected/modules/OphTrOperationbooking/models/Element_OphTrOperationbooking_Operation.php]: Setting the session_id might only be necessary for saving, not for validation. It does seem a bit weird though.

**Review by Vegemash — APPROVED** [review]

**Pull request merged**